bin/compile

```diff
@@ -400,6 +407,21 @@ export GOPATH
 
 # GB installation
 case "${TOOL}" in
+    gomodules)
+        pkgs=${GO_INSTALL_PACKAGE_SPEC:-$(awk '{ if ($1 == "//" && $2 == "+heroku" && $3 == "install" ) { print substr($0, index($0,$4)) }}' ${goMOD})}
```

Comment:
Should we exit `awk` after the first match? If users mistaken have multiple lines of `// +heroku install ./cmd/...` there will be multiple returns:

```
awk '{ if ($1 == "//" && $2 == "+heroku" && $3 == "install" ) { print substr($0, index($0,$4)); exit; }}' ${goMOD}
```